wordpress-mobile/WordPress-Android
# Fixed wrong appbar theme in post list and media picker.

Fixes #12975 

This PR fixes the color of the toolbar icons in the Post List. I also preemptively fixed this issue in media picker activity (gif or stock image picker), even though it doesn't have any icons (toolbar there looks weird, and was fixed in another PR).

To test:
- Navigate to the Post List and make sure the toolbar icons (search and list type) are visible in light and dark modes.

PR submission checklist:
